src/main/kotlin/lindar/acolyte/util/NumberFormatAcolyte.kt

```diff
@@ -61,13 +60,30 @@ class NumberFormatAcolyte {
         return this
     }
 
+    @Deprecated("This function is deprecated. Use showFractionUnitBelowOne() instead.")
     fun showPenniesBelowOne(): NumberFormatAcolyte {
-        showPenniesBelowOne = true
+        showFractionUnitBelowOne = true
         return this
     }
 
+    @Deprecated("This function is deprecated. Use hideFractionUnitBelowOne() instead.")
     fun hidePenniesBelowOne(): NumberFormatAcolyte {
-        showPenniesBelowOne = false
+        showFractionUnitBelowOne = false
```

Comment:
small comment, since this is deprecated and replaced with `showFractionUnitBelowOne()` I think it's implementation should be just `return this. showFractionUnitBelowOne()` to avoid code duplication, and maybe duplicate changes (that might be forgotten) in the future.